paasta_tools/tron_tools.py

```diff
@@ -191,6 +191,13 @@ def pick_spark_ui_port(service, instance):
     return preferred_port
 
 
+def get_smart_paasta_instance_name():
+    if os.environ.get("TRON_JOB_NAMESPACE"):
+        tron_job = os.environ.get("TRON_JOB_NAME")
+        tron_action = os.environ.get("TRON_ACTION")
+        return f"{tron_job}.{tron_action}"
+
```

Comment:
From the tron guide https://github.com/Yelp/Tron/blob/master/docs/command_context.rst, 
these  ENV variables should be available here as well.

Since this pathway is tron only, I'm not checking for other args here